.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,20 @@
-### What changes were proposed in this pull request?
+### What is the goal of this PR?
```

Comment:
Hmm, maybe it's personal but I feel there is some overlap between the 'goal' of a PR, and why the changes in the PR are needed. Quite generally, the goal would be to achieve some requirements. And these requirements are usually part of the why?

Reply:
@vofque Then I would change the first header to be simply "Description". WDYT?

Reply:
I made the changes. WDYT?

Reply:
@agemooij "Description" sounds good to me.